src/model.ts

```diff
@@ -249,8 +247,7 @@ export class Model extends EventBus<any> implements CommandDispatcher {
     this.session.loadInitialMessages(stateUpdateMessages);
 
     for (let Plugin of coreViewsPluginRegistry.getAll()) {
-      const plugin = this.setupUiPlugin(Plugin);
-      this.coreViewsPlugins.push(plugin);
+      const plugin = this.setupCoreUiPlugin(Plugin);
```

Comment:
I didn't think `coreViewPlugin` was a very good name, but IMO CoreUIPlugin is even more confusing.

Anyway the registry should also be renamed if we want to change the nomenclature